### ISSUE TITLE
Watchtower cannot pull images anymore "401 Unauthorized"

### ISSUE THREAD

**Issue by NeoTrace82** [containrrr/watchtower#1081]
Running Watchtower on ARM 64 Raspbian, has been working ever since and stopped now with the following error message:

```
Could not do a head request for "sha256:e1688bbea26473c306345275b9b1f2ccad6c9293af9298d28098c629a77fcbf6", falling back to regular pull.
Reason: registry responded to head request with "401 Unauthorized", auth: "Bearer realm=\"https://auth.docker.io/token\",service=\"registry.docker.io\",scope=\"repository:library/sha256:pull\",error=\"insufficient_scope\""
Unable to update container "/watchtower": Error response from daemon: pull access denied for sha256, repository does not exist or may require 'docker login': denied: requested access to the resource is denied. Proceeding to next.
```

nothing was changed in the config and i do run watchtower every 5 days, so pull request shouldn't be an issue

anyone having or had that issue and knows what to do?

watchtower config:

```
version: "2"
services:
  watchtower:
    image: containrrr/watchtower:latest
    container_name: watchtower
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
    command: --schedule "0 0 4 * * *"
    environment:
      - TZ=Europe/Vienna
      - WATCHTOWER_DEBUG=false
      - WATCHTOWER_NOTIFICATIONS_LEVEL=debug
      - WATCHTOWER_CLEANUP=true
      - DOCKER_API_VERSION=1.24
      - WATCHTOWER_INCLUDE_STOPPED=true
      #- WATCHTOWER_POLL_INTERVAL=13200
      #- WATCHTOWER_POLL_INTERVAL=10
      - WATCHTOWER_NO_PULL=false
      - WATCHTOWER_MONITOR_ONLY=false
      - WATCHTOWER_NOTIFICATIONS=email

      - WATCHTOWER_NOTIFICATION_EMAIL_DELAY=2
      - WATCHTOWER_NOTIFICATION_EMAIL_SUBJECTTAG=DONTPANIC
    restart: unless-stopped
```

**Comment by piksel** [comment]
The actual error is that the image name is just a hash (`sha256:e1688bbea26473c306345275b9b1f2ccad6c9293af9298d28098c629a77fcbf`) which means that watchtower cannot update it. See #1050

**Comment by NeoTrace82** [comment]
> The actual error is that the image name is just a hash (`sha256:e1688bbea26473c306345275b9b1f2ccad6c9293af9298d28098c629a77fcbf`) which means that watchtower cannot update it. See #1050

can't run "latest-dev" as it is only available for amd64 for now, but ill watch #1050 
thanks

**Comment by piksel** [comment]
Closing as duplicate of #1050
